Mathlib/LinearAlgebra/CliffordAlgebra/Even.lean

```diff
@@ -210,27 +210,33 @@ def aux (f : EvenHom Q A) : CliffordAlgebra.even Q →ₗ[R] A := by
 
 @[simp, nolint simpNF] -- Added `nolint simpNF` to avoid a timeout #8386
 theorem aux_one : aux f 1 = 1 :=
-  congr_arg Prod.fst (foldr_one _ _ _ _)
+  letI : AddCommGroup (S f) := AddSubgroupClass.toAddCommGroup _
```

Comment:
If we're going to put these in master, we should add a comment indicating that they are needed by `nightly-testing` as of `2024-04-27` or similar.

My inclination would be to only add these on nightly-testing, as this feel like a hack for something that optimistically might end up fixed before 4.8 is released. I'll let @semorrison, this is ultimately a decision about how the nightly procedure should deal with workarounds, and is not really specific to clifford algebras.

Reply:
Okay, let's wait and see how things shake out of the next few days.